src/node/desktop/test/unit/core/file-path.test.ts

```diff
@@ -603,6 +603,23 @@ describe('FilePath', () => {
       assert.strictEqual(cPath.completeChildPath('../bar'), cPath);
       assert.strictEqual(cPath.completeChildPath('/path/to/quux'), cPath);
     });
+    it('Paths only contain forward slashes with no duplicates', () => {
+      const paths = ['c:\\www\\app\\my/folder/file.r',
+        'C:\\R\\4.1.2\\bin\\\\R.exe',
+        'c:\\\\www\\\\app\\my/folder/file.r'];
+      paths.forEach((path) => {
+        assert.isTrue(path.includes('\\'), `Path ${path} should contain at least a single backward slash for this test to be valid`);
+        const threatedPath = normalizeSeparatorsNative(path);
```

Comment:
Not sure what this variable name means, specificially the use of "threat".

Reply:
I meant treat! Gonna also fix this.